qa/standalone/osd/osd-rep-recov-eio.sh

```diff
@@ -119,12 +119,24 @@ function rados_get_data() {
     COUNT=$(ceph pg $pgid query | jq '.info.stats.stat_sum.num_objects_repaired')
     test "$COUNT" = "1" || return 1
 
+    local object_osds=($(get_osds $poolname $objname))
+    local primary=${object_osds[0]}
+    local bad_peer=${object_osds[1]}
```

Comment:
@xiexingguo Just an FYI.  There is a get_primary() and get_not_primary() ceph-helpers.sh function.  The get_primary() is better than assume the first osd is the primary.

Reply:
@dzafman That's right.

However, considering the `inject_eio` function is already assuming the first osd is the primary:
```
    rados_put $dir $poolname $objname || return 1
    inject_$inject rep data $poolname $objname $dir 0 || return 1
    rados_get $dir $poolname $objname || return 1

    COUNT=$(ceph pg $pgid query | jq '.info.stats.stat_sum.num_objects_repaired')
    test "$COUNT" = "1" || return 1 
```
e.g., the above test would fail if 0 is not primary, can we leave this as it is to keep pace with that?